source/infrastructure/security/ssl-certificates/index.html.md.erb

```diff
@@ -7,4 +7,137 @@ review_in: 6 months
+```
+
+And confirm the answers interactively.
+
+### Request new certificate in Globalsign
+- Open [Globalsign](https://gcc.globalsign.com/au/signon/gacp/usernamePasswordLogin.do)
+- Select _Managed SSL_
+- Choose _Order Certificate_ from either _Department for Education_ or _Education & Skills Funding Agency_
+- Paste the CSR created above
+- Check this CSR, Continue
+- Products: OrganizationSSL
+- SSL Certificate Type: Single Domain Certificate
+- Point of contact must have a team email email address, not an individual
+- Tick box _Is this the Point of Contact for communications?_
+- Continue until `Complete`
+
+The _Platform support_ team will receive the request, validate and you will receive an email with the certificate in PEM (.cer) format.
+
+### Convert to pfx format
+
+It may be required to convert to pfx format, especially in Azure. It replaces pem certificate+private key with pfx certificate+password.
+
+Using openssl:
+
+```
+openssl pkcs12 -export -out cert.pfx -inkey privatekey.key -in cert.cer
+```
+
+## Automatic via key vault
+
+[Azure key vault](https://docs.microsoft.com/en-us/azure/key-vault/certificates/tutorial-rotate-certificates) can generate, renew and store the certificate automatically.
+
+A Globasign administrator account is required and must be approved on case by case basis. It can be requested via the
+[Non-standard / Any Other Request](https://dfe.service-now.com.mcas.ms/serviceportal?id=sc_cat_item&sys_id=3ab186f8db2c2b403b929334ca961998&sysparm_category=19d07bc3dbff17003b929334ca9619bd)
+service portal form. The business service is _Shared IT core services_ and the offering is _SSL Certificate Authority Systems (external)_.
+
+- Configure the Globalsign Certificate Authority (CA) in key vault using the Globalsign admin account. The contact details should be for a team, not an individual.
+- Create a certificate
+  - Method: Generate
+  - Name: Unique name (usually the subdomain)
+  - Type of CA: integrated
+  - CA: the CA created above
+  - Subject (example):
+
+  ```
+C=GB, ST=Greater Manchester, L=Manchester, O=Department for Education, CN=claim-additional-teaching-payment.service.gov.uk
+  ```
+  - DNS Names: list of additional domain names (SANs) to be covered by this certificate
+  - Validity: 12 months
+  - Content type: PKCS #12
+  - Lifetime action type: Automatically renew at a given number of days before expiry
+  - Number of days: 30
+- Click Create
+- Click Certificate Operation and wait for result
+
+## Renewals
+
+The certificates are monitored and owners are notified via email before expiry by either GlobalSign or the _Platform support_ team.
+
+## Check certificate
+
+In order to view certificate expiry details such as expiry date etc, you can run the following command to download the certificate currently being used on a website. In this example, we are focusing on _https://find-postgraduate-teacher-training.education.gov_.uk :
```

Comment:
Now I am being so picky but I havent had coffee :)

etc should be etc. and therefore you should have etc.,

Reply:
fixed thanks